utils/json/src/sector.rs

```diff
@@ -4,20 +4,22 @@
 pub mod json {
     use base64::{prelude::BASE64_STANDARD, Engine};
     use cid::Cid;
-    use fvm_shared::sector::{
-        PoStProof, RegisteredPoStProof, RegisteredSealProof, SectorInfo, SectorNumber,
+    use forest_shim::{
+        sector::{RegisteredPoStProof, RegisteredSealProof, SectorInfo},
+        Inner,
     };
+    use fvm_shared::sector::SectorNumber;
     use serde::{de, Deserialize, Deserializer, Serialize, Serializer};
 
     /// Wrapper for serializing a `PoStProof` to JSON.
     #[derive(Deserialize, Serialize)]
     #[serde(transparent)]
-    pub struct PoStProofJson(#[serde(with = "self")] pub PoStProof);
+    pub struct PoStProofJson(#[serde(with = "self")] pub forest_shim::sector::PoStProof);
```

Comment:
Why not bring `forest_shim::sector::PoStProof` into scope?

Reply:
done